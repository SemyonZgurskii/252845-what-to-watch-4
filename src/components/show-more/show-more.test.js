import ShowMore from './show-more.jsx';
import React from 'react';
import renderer from 'react-test-renderer';

describe(`Create ShowMore snapshot`, () => {
  it(`Should correctly render ShowMore`, () => {
    const showMoreElement = renderer.create(
        <ShowMore
          onClick={() => {}}
        />
    ).toJSON();

    expect(showMoreElement).toMatchSnapshot();
  });
});
