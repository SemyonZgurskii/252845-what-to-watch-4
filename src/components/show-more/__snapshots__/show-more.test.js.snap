// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create ShowMore snapshot Should correctly render ShowMore 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
